gobcore/model/gobmodel.json

```diff
@@ -3806,5 +3806,490 @@
+          },
+          "heeft_verblijfsobject": {
+            "type": "GOB.Reference",
+            "ref": "bag:verblijfsobjecten",
+            "description": "Het verblijfsobject (landelijke identificatie) dat door de nummeraanduiding wordt aangeduid."
+          },
+          "heeft_nummeraanduiding": {
+            "type": "GOB.Reference",
+            "ref": "bag:nummeraanduidingen",
+            "description": "De unieke identificatie/aanduiding van een nummeraanduiding. Een nummeraanduiding is een door het bevoegde gemeentelijke orgaan als zodanig toegekende aanduiding van een adresseerbaar object. De Identificatiecode nummeraanduiding is een combinatie van een viercijferige gemeentecode, de tweecijferige objecttypecode 20 die aangeeft dat het om een nummeraanduiding gaat en een voor het betreffende objecttype binnen een gemeente uniek tiencijferig volgnummer."
+          },
+          "verblijft_in_land": {
+            "type": "GOB.Reference",
+            "ref": "brp:landen",
+            "description": "Een code, opgenomen in Tabel34, Landentabel, die het land (buiten Nederland) aangeeft alwaarde ingeschrevene verblijft. Bij het ingaan van een Ministerieel besluit dient hier de standaardwaarde opgenomen te worden."
+          },
+          "buitenland_regel_1": {
+            "type": "GOB.String",
+            "description": "Eerste deel van het adres in het buitenland, met uitzondering van het land."
+          },
+          "buitenland_regel_2": {
+            "type": "GOB.String",
+            "description": "Tweede deel van het adres in het buitenland, met uitzondering van het land."
+          },
+          "buitenland_regel_3": {
+            "type": "GOB.String",
+            "description": "Derde deel van het adres in het buitenland, met uitzondering van het land."
+          },
+          "datum_aanvang_adres_buitenland": {
+            "type": "GOB.IncompleteDate",
+            "Description": "De datum van van aangifte of ambtshalve melding van verblijf op het buitenlands adres."
+          },
+          "aanduiding_gegevens_in_onderzoek": {
+            "type": "GOB.String",
+            "description": "Numerieke waarde (met voorloopnullen) van een groep of element in een categorie van de BRP, waarvan wordt onderzocht of ze onjuist of in strijd met de openbare orde zijn."
+          },
+          "datum_ingang_onderzoek": {
+            "type": "GOB.IncompleteDate",
+            "Description": "De datum van inschrijving in Nederland."
+          },
+          "datum_einde_onderzoek": {
+            "type": "GOB.IncompleteDate",
+            "Description": "De datum van inschrijving in Nederland."
+          },
+          "aantal_keren_in_onderzoek": {
+            "type": "GOB.integer",
+            "Description": "PLUS: Het totaal aantal keren dat deze verblijfplaats in onderzoek was/is."
+          }
+        }
+      },
+      "gemeenten": {
```

Comment:
Waarom niet gelinked met brk_gemeentes, kan me niks anders bedenken dat gemeentecode gelijk is aan identificatie.

Reply:
Klopt, in tabel 33 van brp zit ook nog een `0000` waarde voor onbekend